src/entity/User.ts

```diff
@@ -8,4 +10,33 @@ export class User extends PickleUserBaseEntity {
 
   @Column()
   password!: string;
+
+  @BeforeInsert()
+  async bI() {
+    this.password = await hash(this.password);
+  }
+
+  @BeforeUpdate()
+  async bU() {
+    this.password = await hash(this.password);
+  }
+}
+
+async function hash(password: string) {
+  if (password) {
+    if (
+      password.length < config.password.minLength ||
+      password.length > config.password.maxLength
+    ) {
+      throw new Error(
```

Comment:
We should create our own Error classes, the problem here is we can't always show the text in a multi-language interface. Perhaps we need a standard way for errors like `INVALID_LENGTH`. Compatibility with Apollo in this scheme would be easier I think.